lib/ansible/modules/network/cloudengine/ce_evpn_global.py

```diff
@@ -0,0 +1,225 @@
+#!/usr/bin/python
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+ANSIBLE_METADATA = {'status': ['preview'],
+                    'supported_by': 'community',
+                    'version': '1.0'}
+
+DOCUMENTATION = '''
+---
+module: ce_evpn_global
+version_added: "2.3"
+short_description: Manages global configuration of EVPN.
+extends_documentation_fragment: cloudengine
+description:
+    - Manages global configuration of EVPN.
+author: Zhijin Zhou (@CloudEngine-Ansible)
+notes:
+    - Before configuring evpn_overlay_enable=false, delete other EVPN configurations.
+options:
+    evpn_overlay_enable:
+        description:
+            - configure EVPN as the VXLAN control plane.
```

Comment:
configure- > Configure

Capital letters & full stops for all `descriptions:` please docs.ansible.com/ansible/dev_guide/developing_modules_documenting.html